src/skmatter/decomposition/_kernel_pcovr.py

```diff
@@ -493,12 +480,14 @@ def score(self, X, Y):
         .. math::
 
             \ell=\frac{\operatorname{Tr}\left[\mathbf{K}_{VV} - 2
-            \mathbf{K}_{VN} \mathbf{T}_N (\mathbf{T}_N^T \mathbf{T}_N)^{-1}  \mathbf{T}_V^T
+            \mathbf{K}_{VN} \mathbf{T}_N
+                (\mathbf{T}_N^T \mathbf{T}_N)^{-1} \mathbf{T}_V^T
             +\mathbf{T}_V(\mathbf{T}_N^T \mathbf{T}_N)^{-1}  \mathbf{T}_N^T
             \mathbf{K}_{NN} \mathbf{T}_N (\mathbf{T}_N^T \mathbf{T}_N)^{-1}
             \mathbf{T}_V^T\right]}{\operatorname{Tr}(\mathbf{K}_{VV})}
 
-        The negative loss is returned for easier use in sklearn pipelines, e.g., a grid search, where methods named 'score' are meant to be maximized.
+        The negative loss is returned for easier use in sklearn pipelines, e.g., a
+        grid search, where methods named 'score' are meant to be maximized.
```

Comment:
Will this render correctly?

Reply:
[Looks good to me.](https://scikit-matter--172.org.readthedocs.build/en/172/pcovr.html#skmatter.decomposition.PCovR.score)

![Screenshot 2023-03-28 at 23 00 58](https://user-images.githubusercontent.com/15192133/228365817-a9bc21fc-ad99-415d-a392-5773ed6b1ade.png)